recipes-devtools/vpi/libnvvpi1/0001-vpi-config-allow-to-update-the-installation-prefix.patch

```diff
@@ -0,0 +1,31 @@
+From 453bb5b6b132ad1758ce018309e5b72173716cf7 Mon Sep 17 00:00:00 2001
+From: Ilies CHERGUI <ilies.chergui@gmail.com>
+Date: Mon, 8 Mar 2021 23:29:02 +0000
+Subject: [PATCH] vpi-config: allow to update the installation prefix
+
+Signed-off-by: Ilies CHERGUI <ilies.chergui@gmail.com>
+---
+ usr/share/vpi1/cmake/vpi-config.cmake | 7 ++++++-
+ 1 file changed, 6 insertions(+), 1 deletion(-)
+
+diff --git a/usr/share/vpi1/cmake/vpi-config.cmake b/usr/share/vpi1/cmake/vpi-config.cmake
+index 638a97d..87de6ee 100644
+--- a/usr/share/vpi1/cmake/vpi-config.cmake
++++ b/usr/share/vpi1/cmake/vpi-config.cmake
+@@ -42,7 +42,12 @@ unset(_expectedTargets)
+ 
+ 
+ # The installation prefix configured by this project.
+-set(_IMPORT_PREFIX "/opt/nvidia/vpi1")
++if (DEFINED ENV{TEGRA_SYSROOT})
```

Comment:
Is there a way to do this without having to introduce an extra environment variable? Some already-defined cmake variable we can use?

Reply:
I don't think so, the original code is as follow:

```
...
# The installation prefix configured by this project.
set(_IMPORT_PREFIX "/opt/nvidia/vpi1")

# Create imported target vpi
add_library(vpi SHARED IMPORTED)

set_target_properties(vpi PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
)
...
```
Nvidia use a hardcoded value.

Reply:
> Nvidia use a hardcoded value.

Yes, but other packages that provide cmake modules don't appear to need an extra environment variable for integration into the Yocto cross-build system.